pyrightconfig.json

```
@@ -0,0 +1,60 @@
+{
+  "typeCheckingMode": "basic",
+  // Target the oldest supported version in editors
+  // "pythonVersion": "3.7", // https://github.com/microsoft/pyright/issues/7330
```

```suggestion
  "pythonVersion": "3.7",
```
https://github.com/microsoft/pyright/releases/tag/1.1.352